### PR TITLE
Create date

### PR DESCRIPTION
**Pull request by SowmyaAji**
@Ardrasal this pull request will add the load_posts file and the html changes that I made to index to accommodate this data. Would you please be kind enough to review?

### CODE REVIEW

File: linkme/linkme/settings.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
     # thirdpartyapps
     'registration',
     'django_extensions',
+    'mimesis',
```
Comment (Ardrasal) [inline linkme/linkme/settings.py]: Nice!

File: linkme/linkworld/templates/index.html [hunk 2]
```diff
@@ -9,6 +9,21 @@
 {% endblock title %}
 
 {% block content %}
- <p>random content</p>
+
+{% for post in posts %}
+  <h3> <a href = "{{ post.title }}">{{ post.title }}</a></h3>
+<<<<<<< Updated upstream
```
Comment (Ardrasal) [inline linkme/linkworld/templates/index.html]: @SowmyaAji please resolve merge conflicts :)

File: linkme/linkworld/templates/index.html [hunk 3]
```diff
@@ -9,6 +9,21 @@
...
+   
+=======
+   <p><a href="{{ post.url }}">link</a></p>
+>>>>>>> Stashed changes
```
Comment (Ardrasal) [inline linkme/linkworld/templates/index.html]: @SowmyaAji  please remove

File: linkme/linkme/settings.py (hunk 1 above)
Comment (SowmyaAji) [reply linkme/linkme/settings.py]: thanks

File: linkme/linkworld/templates/index.html (hunk 2 above)
Comment (SowmyaAji) [reply linkme/linkworld/templates/index.html]: WAIT lemme see

**Pull request merged**